driver/opentelekomcloud_test.go

```diff
@@ -155,3 +162,44 @@ func cleanupResources(driver *Driver) error {
 	}
 	return driver.deleteVPC()
 }
+
+func TestDriver_CreateWithExistingSecGroups(t *testing.T) {
+	preDriver, err := defaultDriver()
+	require.NoError(t, err)
+	require.NoError(t, preDriver.initCompute())
+	newSG := services.RandomString(10, "nsg-")
+	sg, err := preDriver.client.CreateSecurityGroup(newSG, 24)
+	assert.NoError(t, err)
+
+	driver, err := newDriverFromFlags(
+		map[string]interface{}{
+			"otc-cloud":       "otc",
+			"otc-subnet-name": subnetName,
+			"otc-vpc-name":    vpcName,
+			"otc-sec-groups":  sg.Name,
+		})
+	require.NoError(t, err)
+	require.NoError(t, driver.initCompute())
+	require.NoError(t, driver.initNetwork())
+	defer func() {
+		if err := cleanupResources(driver); err != nil {
```

Comment:
`defer func() {`
		`assert.NoError(t, cleanupResources(driver))`
	`}()`
Will we have 2 defer statement styles?

Reply:
Fixed